### ISSUE TITLE
Release 1.6.0 failed to build

### ISSUE THREAD

**Issue by kbestak** [DenisSch/Quantification#50]
The most recent release has a failed Docker image build so the added functionality cannot be used with the Docker image - especially in the context of nf-core.

**Comment by ArtemSokolov** [comment]
Thanks for catching this, @kbestak.
I'll debug this weekend.

**Comment by jmuhlich** [comment]
The immediate/final error is `setuptools-scm was unable to detect version for /src.` This would happen if you had a bare source dump instead of a git checkout, but that doesn't make any sense. Looking farther up the log, I also saw this:
```
#1 [internal] load git source https://github.com/labsyspharm/quantification.git#45d06fd880f9924c1e221cf4bdc72c7c1758f3ad
#1 0.093 Initialized empty Git repository in /var/lib/buildkit/runc-overlayfs/snapshots/snapshots/1/fs/
#1 0.113 fatal: Not a valid object name 45d06fd880f9924c1e221cf4bdc72c7c1758f3ad^{commit}```
```
But that commit `45d06f...` is definitely the commit hash for 1.6.0.

**Comment by jmuhlich** [comment]
Fixed in 1d941840fa4ddb17fd6435cf1ad630ee211e86ae.
